File: app/src/main/java/com/dhsdevelopments/potato/clientapi/notifications/MessageNotification.java
package com.dhsdevelopments.potato.clientapi.notifications;

import com.dhsdevelopments.potato.clientapi.message.Message;
import com.google.gson.annotations.SerializedName;

public class MessageNotification extends PotatoNotification
{
    @SerializedName( "c" )
    public Message message;
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/notifications/MessageNotification.kt
package com.dhsdevelopments.potato.clientapi.notifications

import com.dhsdevelopments.potato.clientapi.message.Message
import com.google.gson.annotations.SerializedName

class MessageNotification : PotatoNotification() {
    @SerializedName("c")
    lateinit var message: Message
}
